src/main/java/com/example/employee/EmployeeService.java

```diff
@@ -0,0 +1,63 @@
+package com.example.employee;
+
+import com.example.employee.dao.Employee;
+import com.example.employee.dto.EmployeeRequest;
+import com.example.employee.dto.EmployeeResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.stream.Collectors;
+
+public class EmployeeService {
+
+    public String addEmployee(HttpServletRequest req) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+
+        String requestBody = req.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
+
+        Employee.save(mapper.readValue(requestBody, EmployeeRequest.class));
+
+        return requestBody;
+    }
+
+    public String getEmployee(HttpServletRequest req) throws JsonProcessingException {
+        ObjectMapper mapper = new ObjectMapper();
+
+        String pathInfo = req.getPathInfo();
+
+        if (pathInfo != null) {
+            String[] pathParts = pathInfo.split("/");
+            EmployeeResponse employee = Employee.getEmployeeById(pathParts[1]);
+
+            if (employee != null) {
+                return mapper.writeValueAsString(employee);
+            }
+        }
```

Comment:
Instead of parsing the url everywhere, create a separate method that does the same.